### PR TITLE
added functional code review generator

### PR DESCRIPTION
**Pull request by pbennettwa**

### CODE REVIEW

File: code-review-randomizer.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// run this script after you do a pull request and group slack the individuals listed
+
+const codeReview = yourName => {
+  const seniorEngineers = ['Inna', 'Kyle', 'Matthew', 'Patrick', 'Sayer']
```
Comment (inkaivasyuk) [inline code-review-randomizer.js]: I think you forgot a ";"

Comment (itsOrD) [reply code-review-randomizer.js]: Great catch!

File: code-review-randomizer.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    if (peopleToReviewYourCode.length < 2) {
+      randomizer();
+    }
+  }
+
+  randomizer();
+  return peopleToReviewYourCode;
+}
+
+codeReview('yourName')
```
Comment (itsOrD) [inline code-review-randomizer.js]: looks like style guide wants there to be an empty line at the bottom of the code block

**Review by itsOrD — CHANGES_REQUESTED** [review]
Need a blank line 22

**Review by kmorgan8588 — COMMENTED** [review]
Looks like a real randomizer to me

**Comment by inkaivasyuk** [comment]
Nice job, Patrick.

**Pull request merged**